lib/rpmsg/rpmsg_virtio.c

```diff
@@ -630,6 +642,13 @@ int rpmsg_init_vdev(struct rpmsg_virtio_device *rvdev,
 	rdev->ops.send_offchannel_nocopy = rpmsg_virtio_send_offchannel_nocopy;
 	role = rpmsg_virtio_get_role(rvdev);
 
+	if (config) {
+		rvdev->config = *config;
+	} else {
+		rvdev->config.txbuf_size = RPMSG_BUFFER_SIZE;
+		rvdev->config.rxbuf_size = RPMSG_BUFFER_SIZE;
```

Comment:
Probably not but should we think about adding `RPMSG_BUFFER_RX_SIZE` and `RPMSG_BUFFER_TX_SIZE` as compile-time defines since we are now decoupling TX and RX?

Reply:
I think if anyone needs it, it could be implemented in other patch